sh/start-stop-daemon.sh

```diff
@@ -38,10 +38,6 @@ ssd_start()
 		service_inactive && _inactive=true
 		mark_service_inactive
 	fi
-	[ -n "$output_logger" ] &&
-		output_logger_arg="--stdout-logger \"$output_logger\""
-	[ -n "$error_logger" ] &&
-		error_logger_arg="--stderr-logger \"$error_logger\""
```

Comment:
This change is not related to the topic of this commit/PR, i.e. shouldn’t be here.